tests/units/Model/UserModelTest.php

```diff
@@ -401,6 +401,5 @@ public function testTrimUsername()
         $this->assertNotFalse($userModel->create(array('username' => 'test ')));
 
         $this->assertNotEmpty($userModel->getByUsername('test'));
-        $this->assertEmpty($userModel->getByUsername('test '));
```

Comment:
Hi @fguillot,

I put it in the commit message, but apparently GitHub didn't include it here... :(  This is what I said:

> Microsoft SQL Server automatically pads the shorter of two strings of
> unequal length with spaces prior to comparisons. Thus, searching for
> username = 'test ' (unexpectedly) matches a row where username = 'test'.
> 
> Since kanboard does not rely on this behavior or its absence, simply
> remove the test as unecessary to avoid this differing result amongst the
> various database backends.
> 
> Ref: https://support.microsoft.com/en-us/topic/inf-how-sql-server-compares-strings-with-trailing-spaces-b62b1a2d-27d3-4260-216d-a605719003b0